src/status/status.factory.js

```diff
@@ -34,19 +51,35 @@ module.exports = (logProvider, logger) => {
     return memeberStatus;
   }
 
+  function eventMemberStatus(events) {
```

Comment:
can you rename this to something like 'allTimeEventStatus' to keep it consistent with 'allTimeMemberStatus' or you could rename them both to something else that is more explicit